frontend-shared/README.md

```diff
@@ -0,0 +1,37 @@
+# Shared resources for Hypothesis front-end applications
+
+A package of resources for Hypothesis front-end applications.
+
+#### Requirements
+
+- preact
+- browserify
```

Comment:
browserify is required because of the svg files. They need to be stringified. The package.json has the browserify config that the client can consume. It's a bit of odd coupling that we will have to work out when moving to its own repo.